app/src/controllers/docs.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+  'use strict';
+  angular.module('prodocs.controllers')
+    .controller('DocsCtrl', ['$scope', '$mdMedia',
+      '$timeout', 'Docs',
+      function($scope, $mdMedia, $timeout, Docs) {
```

Comment:
'Docs' is defined but never used.<br>'$timeout' is defined but never used.<br>'$mdMedia' is defined but never used.<br>'$scope' is defined but never used.